desktop/core/src/desktop/js/apps/storageBrowser/StorageDirectoryPage/CreateAndUploadAction/CreateAndUploadAction.tsx

```diff
@@ -151,24 +151,28 @@ const CreateAndUploadAction = ({
           <DropDownIcon />
         </PrimaryButton>
       </Dropdown>
-      <InputModal
-        title={t('Create New Folder')}
-        inputLabel={t('Enter New Folder Name')}
-        submitText={t('Create')}
-        showModal={selectedAction === ActionType.createFolder}
-        onSubmit={handleCreate}
-        onClose={onModalClose}
-        buttonDisabled={loading}
-      />
-      <InputModal
-        title={t('Create New File')}
-        inputLabel={t('Enter New File Name')}
-        submitText={t('Create')}
-        showModal={selectedAction === ActionType.createFile}
-        onSubmit={handleCreate}
-        onClose={onModalClose}
-        buttonDisabled={loading}
-      />
+      {selectedAction === ActionType.createFolder && (
+        <InputModal
+          title={t('Create New Folder')}
+          inputLabel={t('Folder name')}
```

Comment:
is it mostly same for both inputs expect the title
why not `title={selectedAction===ActionType.createFolder?t('Folder name'): t('File name')}`